allennlp/interpret/influence_interpreters/influence_interpreter.py

```diff
@@ -0,0 +1,109 @@
+from typing import List, Optional
+
+import torch
+
+from allennlp.common import Registrable
+from allennlp.common.util import JsonDict
+from allennlp.models.model import Model
+from allennlp.predictors import Predictor
+from allennlp.data.dataset_readers import DatasetReader
+from allennlp.data.data_loaders import MultiProcessDataLoader
+
+
+class InfluenceInterpreter(Registrable):
```

Comment:
So, how does one actually use this class since it has no methods?

I think there should be a method like this:

```python
def interpret(self, train_data_path, test_data_path):
    pass
```

And then there's no need to pass `train_filepath` and `test_filepath` into `__init__`.

This makes it clear how the class should be used, and how subclasses should implement it. And also allows an instance of this class to be used multiple times, on different test data paths.

Reply:
I think that makes sense. I will make the change. However, I think it's necessary to specify the `train_file` in __init__ right? The train source should be invariant; giving freedom to change test examples does sound useful

Reply:
Yea, that sounds reasonable